guideline-11.md

```diff
@@ -1,9 +1,21 @@
+Preventing users from completing desired tasks via constant alerts, messages, and advertisements is prohibited behavior. When plugins get between users and their work it creates a poor experience. Overwhelming the admin dashboard to the point of where it is no longer usable is detrimental to WordPress as a whole.
 
-Upgrade prompts, notices, alerts, and the like must be limited in scope and used sparingly, be that contextually or only on the plugin’s setting page. Site wide notices or embedded dashboard widgets <em>must</em> be dismissible or self-dismiss when resolved. Error messages and alerts must include information on how to resolve the situation, and remove themselves when completed.
+At the same time, in order to support the plugin ecosystem it is permitted for plugins to display messages as to their functionality, as well as to promote included features. These messages must be done with consideration and care towards usability. All messages must display only for the appropriate users (i.e. people who can action on them).
 
-Advertising within the WordPress dashboard should be avoided, as it is generally ineffective. Users normally only visit settings pages when they're trying to solve a problem. Making it harder to use a plugin does not generally encourage a good review, and we recommend limiting any ads placed therein. Remember: tracking referrals via those ads is not permitted (see guideline 7) and most third-party systems do not permit back-end advertisements. Abusing the guidelines of an advertising system will result in developers being reported upstream.
+Any messages added by a plugin must be <strong>permanently</strong> dismissible. This may be done either by a user's direct action (clicking a 'dismiss' button) or by self-removal when an issue is resolved. Messages are to be limited in scope to one per page, with an exception granted to a plugin's individual settings page(s).
 
-Developers are welcome and encouraged to include links to their own sites or social networks, as well as locally (within the plugin) including images to enhance that experience.
\ No newline at end of file
+Included advertisements for add-ons, upsells, or services must <em>directly</em> relate to the plugin and/or developer. Affiliate links must go directly to the affiliate and not a 3rd party URL shortener.
+
+Examples of violations include (but are not limited to):
+
+<ul>
+    <li>Messages that cannot be dismissed, nor resolved</li>
+    <li>Messages that repeat regularly and cannot be permanently dismissed</li>
+    <li>Multiple messages on any one page, other than a plugin's own setting page(s)</li>
+    <li>Advertisements for unrelated products (such as a favored forms plugin when it has no relationship to the active plugin)</li>
+    <li>Hidden affiliate links to other services (such as hosting, Amazon, etc)</li>
+</ul>
+
+Abuse of the notifications system often result in poor reviews from users. Those reviews will not be removed.
```

Comment:
Is there more documentation around when reviews _are_ removed and for what reasons? If so, does it make sense to link to that here? If not, does it make sense to write that (eventually?)

Reply:
That's actually handled by the support team :) @clorith is working on a revamp of those guidelines!